crypto/err/err.c

```diff
@@ -654,18 +614,9 @@ const char *ERR_lib_error_string(unsigned long e)
 
 const char *ERR_func_error_string(unsigned long e)
 {
-    ERR_STRING_DATA d, *p;
-    unsigned long l, f;
-
-    if (!RUN_ONCE(&err_string_init, do_err_strings_init)) {
+    if (!RUN_ONCE(&err_string_init, do_err_strings_init))
         return NULL;
-    }
-
-    l = ERR_GET_LIB(e);
-    f = ERR_GET_FUNC(e);
-    d.error = ERR_PACK(l, f, 0);
-    p = int_err_get_item(&d);
-    return ((p == NULL) ? NULL : p->string);
+    return ERR_GET_LIB(e) == ERR_LIB_SYS ? "system library" : "internal";
```

Comment:
I would rather have it return NULL

Reply:
It breaks things if you return NULL.

Reply:
Do you mean it breaks `ERR_error_string_n`? Our discussion on printing function indicators might resolve that issue.

Reply:
I know it breaks that function, as I recall while doing some test/dev on this PR something else broke but I don't recall specifics.

Reply:
Considering the code before your change *could* return NULL, it seems the callers are assuming too much.

Reply:
changed.